test/membrane/globals.spec.js

```diff
@@ -66,23 +66,45 @@ describe('sandbox', () => {
             expect(Object.getOwnPropertyDescriptors.length).toBe(1);
             expect(Object.getOwnPropertyDescriptors.toString()).toContain('[native code]');
 
-            // Chrome has a JIT bug when working with a detached iframe and
```

Comment:
can someone provide more details of the actual bug?
maybe a ref or something? cc @rwaldron @jdalton

Reply:
I got you. With the removal of that comment I outlined in more detail in two places in our code base linking to the bug report in both places. The bug filed is https://bugs.chromium.org/p/chromium/issues/detail?id=1305302

Reply:
The gist is
```js
// Chrome has a bug when working with detached iframes and their window
// getters.
// https://bugs.chromium.org/p/chromium/issues/detail?id=1305302
//
// Some methods are poisoned when they interact with the window object
// and retrieve property descriptors, like 'window', that contain the
// window object itself. Once retrieved, the window getter will return
// `null` instead of the window object. The following methods run into
// this issue:
//     console.log(window);
//     Object.getOwnPropertyDescriptors(window);
//     Object.getOwnPropertyDescriptor(window, 'globalThis');
//     Reflect.getOwnPropertyDescriptor(window, 'globalThis');
//
// We side step issues with console by mapping it to the the blue
// realm's console. Since we're already wrapping GetOwnPropertyDescriptor(s)
// methods to activate lazy descriptors we use the wrapper to workaround
// the window nulling bug.
```